cli/userInfo.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/hashicorp/vault/api"
+)
+
+// Connects to Vault server
+func Client(token string) (*api.Client, error) {
```

Comment:
Code handling the connection to Vault should not be in a file called `userInfo.go` (nor `handlers.go`). `vault.go` would be a better location for it.